data/fl/retired/Olga-Gonzalez-0064c658-0bb3-4855-be13-5d91d69d0cf0.yml

```diff
@@ -0,0 +1,15 @@
+id: ocd-person/0064c658-0bb3-4855-be13-5d91d69d0cf0
+name: Olga Gonzalez
```

Comment:
Not sure. Let me fix manually.

Latest update to script last night was to make it so that you can run `syncdown` more idempotently.

Previous versions of the `syncdown` script would create a new `ocd-person-id` if it detected that a person from airtable had an existing id but with a new name. But it wasn't also checking if the new airtable person was already created (on previous syncdown)

I thought I had only run the syncdown one time previously, but maybe I did multiple at some point

Reply:
I think what's happening w Olga Gonzalez is that she's a new mayor and got entered into airtable, but the `end_date` got left as `2020-12-31` so she gets synced down but then immediately retired. 

Let me try to figure out when her new term ends...

Reply:
ok should be fixed now